app/views/candidate_interface/adviser_sign_ups/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.new_adviser_sign_up'), @adviser_sign_up.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: @adviser_sign_up, url: candidate_interface_adviser_sign_ups_path) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">Sign up for a Teacher Training Adviser</h1>
```

Comment:
I'm aware a lot of this content is TBC, but we should ideally be putting them in the i18n files anyway.